server/app/services/export/QuestionJsonPresenter.java

```diff
@@ -166,6 +169,10 @@ public ImmutableMap<Path, Optional<ImmutableList<String>>> getJsonEntries(
               .map(LocalizedQuestionOption::optionText)
               .collect(ImmutableList.toImmutableList());
 
+      if (selectedOptions.isEmpty()) {
+        return ImmutableMap.of(path.asArrayElement(), Optional.empty());
+      }
+
```

Comment:
Instead of returning an empty array to represent the unanswered multiselect question, we return an empty optional at a path with an array element.

This allows us to standardize our handling of both unanswered multi-selects and unanswered enumerator questions.